docker/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM registry.access.redhat.com/ubi7/ubi:latest
+
+MAINTAINER Thingpedia Admins <thingpedia-admins@lists.stanford.edu>
+
+# install basic tools
+RUN yum -y install git wget gcc gcc-c++ make gettext
+
+# add epel repo
+RUN wget http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+RUN yum -y install epel-release-latest-7.noarch.rpm
+
+# install nodejs 10.x and yarn
+RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash -
+RUN curl -sL https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo
+RUN yum -y install nodejs yarn
+
+# install python3
+RUN yum -y install python36 \
+	python36-pip \
+	python36-devel
+
+# install aws client
+RUN pip3 install awscli
+RUN curl --silent --location \
+    "https://github.com/weaveworks/eksctl/releases/download/latest_release/eksctl_$(uname -s)_amd64.tar.gz" | \
+    tar xz -C /usr/local/bin
```

Comment:
we might also need eksctl  and kubectl (in the future using jobs) for k8 GPU training?

Reply:
I expect we will use the API directly, through the [nodejs library](https://github.com/kubernetes-client/javascript)

Reply:
Do you need to  download eksctl?  It's not included in awsclli.